packages/core/test/Form_test.js

```diff
@@ -2195,22 +2301,30 @@ describeRepeated("Form common", createFormComponent => {
         },
       };
 
-      const { comp } = createFormComponent(formProps);
+      const { comp, node, onError } = createFormComponent(formProps);
 
-      expect(comp.state.errorSchema).eql({});
+      submitForm(node);
+      sinon.assert.notCalled(onError);
 
       setProps(comp, {
         ...formProps,
+        onError,
```

Comment:
Do we need this line? (also line 2356)

Reply:
If not, the setProps function will not include an onError spy function that we can assert on. There might be a better way to fix this, but that felt out of scope for this PR.